addTagForm.php

```diff
@@ -0,0 +1,19 @@
+
+<h1>Add a New Tag</h1>
+<main class="general">
+    <form class="addfb-form" method="post">
+        <h2>New Tag</h2>
+        <p>An asterisk (<label><em>*</em></label>) indicates a required field.</p>
+        <fieldset>
+            <legend>Add New Tag</legend>
+            <label for="tag"><em>* </em>Input Tag:</label>
+            <input type="text" id="tag" name="tag" required placeholder="Enter new tag">
+        </fieldset>
+        
+
+        <p>Press submit to add tag.</p>
+        <input type="submit" name="addtag-form" value="Submit">
+    </form>
+        <a class="button cancel" href="index.php" style="margin-top: .5rem">Cancel</a>
```

Comment:
Should be some check if tag name or id matches any other in the database, and cancel if so